vivarium/processes/agent_names.py

```diff
@@ -1,12 +1,14 @@
 from __future__ import absolute_import, division, print_function
```

Comment:
We could go through and remove all these `__future__`` imports as well, since we not longer support Python 2.  Let's add that to the TODO list.

Reply:
Done in d7f9921